swift_package/BrainFlow/Sources/BoardDescription.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  BoardDescription.swift
+//  a Swift reimagining of BrainFlow's board_description data type
+//
+//  Created by Scott Miller for Aeris Rising, LLC on 8/23/21.
+//
+import Foundation
+//import BrainFlow
+
+struct BoardDescription: Codable, Equatable {
+    let package_num_channel: Int32
+    let timestamp_channel: Int32
+    let accel_channels: [Int32]?
+    let ecg_channels: [Int32]?
+    let eeg_channels: [Int32]?
+    let eeg_names: String?
+    let emg_channels: [Int32]?
+    let eog_channels: [Int32]?
+    let marker_channel: Int32?
+    let name: String?
+    let num_rows: Int32?
+    let sampling_rate: Int32?
+    let battery_channel: Int32?
+    let eda_channels: [Int32]?
+    let gyro_channels: [Int32]?
+    let ppg_channels: [Int32]?
+    let resistance_channels: [Int32]?
+    let temperature_channels: [Int32]?
+    
+    // decode the input JSON into self:
+    init(_ descriptionJSON: String) throws {
```

Comment:
it should be possible to create an instance of this class wo json string, default constructor shoult take nothing and set everything to zeroes or empty strings
dont know about swift but in other languages it would be good to have static methods like to/from json

Reply:
Done. I also added a testBoardDescription() function to BrainFlowCItests.